src/main/kotlin/com/viartemev/thewhiterabbit/rpc/RpcOutboundMessage.kt

```diff
@@ -0,0 +1,8 @@
+package com.viartemev.thewhiterabbit.rpc
+
+data class RpcOutboundMessage(
+    val exchangeName: String,
+    val requestQueueName: String,
+    val replyQueueName: String,
+    val message: String
```

Comment:
Сообщение должно быть бинарным чтобы не ограничивать форматы кодирования и данных вообще.
Например я бы хотел иметь возможность использовать cbor или kryo

Reply:
Тогда наверное на publish тоже поменять надо на byte array, вместо string

Reply:
That;s possible. I'm sorry for russian in CR, I was in hurry and completely forgotten that we welcome newcomers to Kotlin, RabbitMQ and coroutines :)

As I said before I don't know how the whole thing works, but that's pretty obvious that AMQP is binary format and it makes sense to send `byte[]`s or `ByteBuffer`s instead of strings to be able to send non-string-encoded messages.

And yes, it makes sense to send binary data on publish too. And on consume too :)